### DISCUSSION TITLE
GoPro Hero 12 - No video and Unable to Update to Labs FW

### DISCUSSION THREAD

**Discussion by flopbuster** [gopro/labs#1011] [Q&A]
Hi everyone,

I hope you're all doing well. I'm having some trouble with my Hero 12 and was hoping for some advice.

My camera was working fine until I tried updating to the latest Labs firmware, following the manual update instructions. The update didn't work, and now I'm facing several issues:

With the SD card containing the Update folder inserted, the engine icon indicating the update start did not appear, and the GoPro kept rebooting repeatedly after a few seconds.
After the first attempt, I removed the SD card, and the GoPro no longer shows any video. It boots up, I can see the menu, but there's no image at all.
It's also acting strangely, like taking around 30 seconds to switch from video to camera mode.
I've tried a factory reset, but the same issues persist. I can't update to Labs, and there's no video.

No idea what to do. Any suggestions on how to fix this?

Thanks a lot for your help!

**Comment by dnewman-gpsw** [comment]
You should contact GoPro's support.  You may I have a camera failure.
